test/labeling/model/test_label_model.py

```diff
@@ -227,26 +227,26 @@ def test_loss(self):
 
         # l2_loss = l2*M*K*||mu - mu_init||_2 = 3*2*(0.05^2) = 0.03
         self.assertAlmostEqual(
-            label_model.loss_l2(l2=1.0).detach().numpy().ravel()[0], 0.03
+            label_model._loss_l2(l2=1.0).detach().numpy().ravel()[0], 0.03
```

Comment:
Nit: `label_model._loss_l2(l2=1.0).item()`

use `.item()` for all below changes